devices.js

```diff
@@ -4816,6 +4816,16 @@ const devices = [
             await configureReporting.acPowerDivisor(endpoint);
         },
     },
+    {
+        zigbeeModel: ['SZ-DWS02'],
+        model: 'SZ-DWS02',
+        vendor: 'Securifi',
+        description: 'Door and Window sensor',
+        supports: 'contact, tamper',
+        fromZigbee: [fz.securifi_contact],
```

Comment:
Please try to use `fz.ias_contact_alarm_1` instead of `fz.securifi_contact`

Reply:
Will make that change and see what I get.